.storybook/preview.js

```diff
@@ -25,37 +27,55 @@ const I18nProvider = (props) => {
   );
 };
 
+function mockFetch(path, init) {
+  if (path === '/api/orgs/1/people/1' && init === undefined) {
+    return Promise.resolve(
+      new Response(
+        JSON.stringify({
+          data: mockPerson(),
+        })
+      )
+    );
+  } else if (path === '/api/orgs/1/people/1/tags' && init === undefined) {
+    return Promise.resolve(new Response('[]'));
+  } else {
+    return Promise.reject({ error: 'unmocked request', path, init });
+  }
+}
+
+class MockApiClient extends FetchApiClient {
+  constructor() {
+    super(mockFetch);
+  }
+}
+
 export const decorators = [
-  muiTheme([theme]),
-  (story) => <ReduxProvider store={createStore()}>{story()}</ReduxProvider>,
+  (Story) => (
+    <ThemeProvider theme={theme}>
+      <Story />
+    </ThemeProvider>
+  ),
+  (story) => {
+    const store = createStore();
+    const router = useRouter();
+    const env = new Environment(store, new MockApiClient(), router);
+    return (
+      <ReduxProvider store={store}>
+        <EnvProvider env={env}>{story()}</EnvProvider>
+      </ReduxProvider>
```

Comment:
I added all the providers but I guess these are the ones that are actually needed! Nice to see your solution :D

Reply:
Yeah, I don't know which is better of `Providers` or my mix of decorators. On `main` we have `ReduxProvider` and `I18nProvider` as separate decorators and I just built on that. I didn't know about `Providers` until I saw your branch.